cabot/cabotapp/models.py

```diff
@@ -249,13 +277,20 @@ def update_status(self):
         help_text="URL of service."
     )
 
+    schedule = models.ForeignKey(
```

Comment:
Changed to ManyToMany (OneToMany is ForeignKey in django) since services can have multiple schedules, and multiple services can have the same schedule.